From 65d932c13d7827c8c531ed85c1e253b4d1466dd3 Mon Sep 17 00:00:00 2001
From: bgill55 <bricam55@gmail.com>
Date: Sun, 25 Jan 2015 15:17:10 -0700
Subject: [PATCH 15/17] revert an uneeded color code exposure in the dialer
 this will fix the switch button color when activated in the General settings
 and other locations.

Change-Id: Iead04fdfe3a02abf8eddf7e0032e3ed8212b96f9
---
 res/values/styles.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/res/values/styles.xml b/res/values/styles.xml
index 1f8be68..69e1471 100644
--- a/res/values/styles.xml
+++ b/res/values/styles.xml
@@ -80,7 +80,7 @@
         <item name="android:colorPrimaryDark">@color/exposed_secondary_text_light</item>
         <item name="android:navigationBarColor">@color/dialer_theme_color_dark</item>
         <item name="dialpad_key_button_touch_tint">@color/dialer_dialpad_touch_tint</item>
-        <item name="android:colorControlActivated">@color/exposed_primary_text_dark</item>
+        <item name="android:colorControlActivated">@color/dialer_theme_color</item>
     </style>
 
     <!-- Action bar overflow menu icon. -->
-- 
2.5.0

